src/js/components/DateInput/DateInput.js

```diff
@@ -396,7 +395,14 @@ Use the icon prop instead.`,
     if (open && !readOnly) {
       return [
         input,
-        <Keyboard key="drop" onEsc={() => ref.current.focus()}>
+        <Keyboard
+          key="drop"
+          onEsc={() => {
+            setTimeout(() => {
+              ref?.current.focus();
+            }, 1);
```

Comment:
Any reason this is 1 rather than 0? I think the goal is just to schedule this immediately after the current thread of execution

Reply:
Updated to use 0 instead of 1